crates/ndc-sendgrid/src/mutation.rs

```diff
@@ -108,19 +96,17 @@ async fn process_send_mail(
             match field {
                 Field::Column { column } => {
                     let field_value = match column.as_str() {
-                        "batch_id" => RowFieldValue(
-                            request
-                                .batch_id
-                                .clone()
-                                .map_or(Value::Null, |id| Value::String(id)),
-                        ),
+                        "status" => RowFieldValue(Value::Object(serde_json::Map::from_iter([(
+                            "status".into(),
+                            Value::String("success".to_string()),
+                        )]))),
                         other_column => {
                             return Err(MutationError::InvalidRequest(format!(
                                 "Unknown column {other_column}"
                             )))
                         }
                     };
-                    row.insert(field_name, field_value);
+                    row.insert("__value".to_string(), field_value);
```

Comment:
This is wrong if we provide more fields for response, but works with only one field.